src/details/ArborX_DetailsBoundingVolumeHierarchyImpl.hpp

```diff
@@ -72,6 +72,35 @@ enum class NearestQueryAlgorithm
   PriorityQueueBased_Deprecated
 };
 
+} // namespace Details
+
+struct TraversalPolicy
+{
+  int _buffer_size = 0;
+  bool _do_predicate_sort = true;
+  Details::NearestQueryAlgorithm _which =
+      Details::NearestQueryAlgorithm::StackBased_Default;
```

Comment:
Name is a bit silly for a data member (not very descriptive)

Reply:
That's fair, I was not thinking, just copy-pasting the original `which` parameter.